corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -327,7 +327,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             this.styles = options.styles;
             this.hasNoItems = options.collection.length === 0;
             this.redoLast = options.redoLast;
-            this.selectedCaseIds = [];
+            this.selectedCaseIds = sessionStorage.selectedValues === undefined || sessionStorage.selectedValues.length === 0 ?  [] : sessionStorage.selectedValues.split(',');
```

Comment:
This could be more concise if you just checked for falsiness instead of both undefined & blank string.

(same comment on the same expression below)